timescaledb/overview/release-notes/index.md

```diff
@@ -71,9 +71,12 @@ exclude all but the most recent chunk were dominated by planning time.
 This new optimization enhances the planner to take the `now()` constraint into
 consideration when planning thereby reducing the work that needs to be done
 in the planner.
+
+This release also contains bug fixes and minor feature improvements since the 2.7.0 release.
+
 <!-- <highlight type="note"> This release is low priority for upgrade. We recommend that you upgrade when you can. </highlight> -->
 
-<highlight type="important"> This release is medium priority for upgrade. We recommend that you upgrade at the next available opportunity. </highlight>
+<highlight type="important"> This release is a patch release. We recommend that you upgrade at the next available opportunity. </highlight>
```

Comment:
These are standard phrases, so we shouldn't really be changing them. Can we add the patch release info elsewhere, and leave this with the original phrasing?

Reply:
Thanks, I've updated as you suggested.